app/presenters/hyrax/manifest_components.rb

```diff
@@ -0,0 +1,78 @@
+module Hyrax
+  module ManifestComponents
+    delegate :rendering_ids, to: :solr_document
+
+    def manifest_url
+      manifest_helper.polymorphic_url([:manifest, self])
+    end
+
+    # IIIF rendering linking property for inclusion in the manifest
+    #
+    # @return [Array] array of rendering hashes
+    def sequence_rendering
+      renderings = []
+      if solr_document.rendering_ids.present?
+        solr_document.rendering_ids.each do |file_set_id|
+          renderings << build_rendering(file_set_id)
+        end
+      end
+      renderings.flatten
+    end
+
+    # IIIF metadata for inclusion in the manifest
+    #
+    # @return [Array] array of metadata hashes
+    def manifest_metadata
+      metadata = []
+      metadata_fields.each do |field|
+        metadata << {
+          'label' => field.to_s.humanize.capitalize,
+          'value' => wrapped_metadata_values(field)
+        }
+      end
+      metadata
+    end
+
+    private
+
+      def manifest_helper
+        @manifest_helper ||= ManifestHelper.new(request.base_url)
+      end
+
+      # Build a rendering hash
+      #
+      # @return [Hash] rendering
+      def build_rendering(file_set_id)
+        file_set_document = query_for_rendering(file_set_id).first # only one result is returned
+        label = file_set_document['label_ssi'] ? ": #{file_set_document['label_ssi']}" : ''
+        {
+          '@id' => Hyrax::Engine.routes.url_helpers.download_url(file_set_document[ActiveFedora.id_field], host: request.host),
```

Comment:
can we just use `file_set_id` here?

Reply:
I'm not following?

Reply:
use `file_set_document.id` rather than `file_set_document[ActiveFedora.id_field]` because `ActiveFedora` is going away as a dependency when we add Valkyrie. `::SolrDocument.primary_key` may be even more correct.

Reply:
Done